### PR TITLE
changed obsidian link

### PR DESCRIPTION
**Pull request by Jmack2**

### CODE REVIEW

File: resources/obsidian.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ permalink: /resources/obsidian/
 
 Check out our new testing tool, Obsidian!
 
-visit [www.obsidiantest.org](http://obsidiantest.org)
\ No newline at end of file
+visit <a href="https://github.com/johnnyLadders/obsidian" target="_blank">www.obsidiantest.org</a>
```
Comment (johnzeringue) [inline resources/obsidian.md]: Why aren't you using the Markdown syntax here?

**Comment by Jmack2** [comment]
@johnzeringue the markdown syntax does not allow for opening in a new tab/window, which @bowring would like to have.

**Comment by johnzeringue** [comment]
Have you tried [this](https://github.com/gettalong/kramdown/issues/122)? This is obviously not a huge deal, but it would be nice to find a way to be consistent.

**Comment by Jmack2** [comment]
No I haven't. In all the googling I have done, I was told that something like that was not in the syntax. I'll try it!

**Comment by johnzeringue** [comment]
If you notice in the config, we're using the [kramdown](http://kramdown.gettalong.org) parser, which seems to mean we get the Markdown spec and then some.

**Pull request merged**